### PR TITLE
VIH-6088 display participant network icons horizontally on the vho hearing list

### PR DESCRIPTION
**Pull request by RocheL**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-6088
### Change description ###
 - updated UI to move the heartbeat icons below the status display.
**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss [hunk 1]
```diff
@@ -60,3 +60,79 @@
...
+  background: #fff;
+  width: auto;
+  height: auto;
+  border-radius: 3px;
+  position: absolute;
+  z-index: 99;
+  box-shadow: 0 7px 11px 0px rgba(0, 0, 0, 0.3);
+}
+.boxHead {
+  font-family: 'GDS Transport', Arial, sans-serif;
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss:74]: Could you stick to the GovUK styles i.e. 
@include govuk-font($size: 16, $weight: 'bold');

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss [hunk 2]
```diff
@@ -60,3 +60,79 @@
...
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 1.14286;
+  background: #626a6e;
+  color: #fff;
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss:81]: govuk colours here

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss [hunk 3]
```diff
@@ -60,3 +60,79 @@
   width: 100%;
   margin-top: 30px;
 }
+
+.box {
+  background: #fff;
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss:65]: govuk colour

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss (hunk 1 above)
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss]: replaced with mixins

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss (hunk 2 above)
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss]: updated to govuk-colour

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss (hunk 3 above)
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss]: updated to govuk-colour for all the colours

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss [hunk 4]
```diff
@@ -60,3 +61,91 @@
...
+  height: auto;
+  border-radius: 3px;
+  position: absolute;
+  z-index: 99;
+  box-shadow: 0 7px 11px 0px rgba(0, 0, 0, 0.3);
+}
+
+.boxHead {
+  @include govuk-font($size: 16, $weight: 'bold');
+  -webkit-font-smoothing: antialiased;
```
[the remark this answers is not in the document]
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss]: removed the smoothing css

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss [hunk 5]
```diff
@@ -60,3 +60,79 @@
...
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 1.14286;
+  background: #00703c;
+  color: #fff;
+  padding: 12px 16px;
+  border-radius: 0px 0px 3px 3px;
+}
+.boxFoot--Dead {
+  background: #0b0c0d;
```
[the remark this answers is not in the document]
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss]: updated colour to govuk black

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss [hunk 6]
```diff
@@ -60,3 +60,79 @@
...
+}
+.signalHeadTitle {
+  grid-column: 2/2;
+  justify-self: end;
+  align-self: center;
+}
+.boxBody {
+  font-family: 'GDS Transport', Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```
[the remark this answers is not in the document]
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss]: updated to mixins

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss (hunk 5 above)
[the remark this answers is not in the document]
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss]: updated to govuk('black')

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss [hunk 7]
```diff
@@ -60,3 +60,79 @@
...
+  font-size: 16px;
+  line-height: 1.14286;
+  background: #00703c;
+  color: #fff;
+  padding: 12px 16px;
+  border-radius: 0px 0px 3px 3px;
+}
+.boxFoot--Dead {
+  background: #0b0c0d;
+  color: #fff;
```
[the remark this answers is not in the document]
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss]: update to govuk('white')

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss [hunk 8]
```diff
@@ -60,3 +60,79 @@
...
+  padding: 12px 16px 8px 16px;
+  margin-top: 0px;
+  margin-bottom: 0px;
+}
+.boxFoot {
+  font-family: 'GDS Transport', Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 700;
+  font-size: 16px;
```
[the remark this answers is not in the document]
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss]: updated to mixins

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss [hunk 9]
```diff
@@ -60,3 +60,79 @@
...
+}
+.boxBody {
+  font-family: 'GDS Transport', Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 1.14286;
+  background: #626a6e;
+  color: #fff;
```
[the remark this answers is not in the document]
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss]: updated to govuk('white')

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss [hunk 10]
```diff
@@ -60,3 +60,79 @@
...
+  border-radius: 3px;
+  position: absolute;
+  z-index: 99;
+  box-shadow: 0 7px 11px 0px rgba(0, 0, 0, 0.3);
+}
+.boxHead {
+  font-family: 'GDS Transport', Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
```
[the remark this answers is not in the document]
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss]: updated to mixins

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss [hunk 11]
```diff
@@ -60,3 +60,79 @@
...
+  align-self: center;
+  font-weight: 700;
+}
+.signalHeadTitle {
+  grid-column: 2/2;
+  justify-self: end;
+  align-self: center;
+}
+.boxBody {
+  font-family: 'GDS Transport', Arial, sans-serif;
```
[the remark this answers is not in the document]
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss]: updated to mixins

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss [hunk 12]
```diff
@@ -60,3 +60,79 @@
...
+  box-shadow: 0 7px 11px 0px rgba(0, 0, 0, 0.3);
+}
+.boxHead {
+  font-family: 'GDS Transport', Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 1.14286;
+  background: #626a6e;
```
[the remark this answers is not in the document]
Comment (RocheL) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss]: update to govuk colour

**Pull request merged**